mllib/src/main/scala/org/apache/spark/ml/param/sharedParams.scala

```diff
@@ -34,6 +34,43 @@ private[ml] trait HasRegParam extends Params {
   def getRegParam: Double = get(regParam)
 }
 
+private[ml] trait HasElasticNetParam extends HasRegParam {
```

Comment:
I think the names should definitely match, so we should pick one.  I'm OK with "elasticNetParam" or "regAlpha."  (I think "regAlpha" doesn't need "Param" attached to it since "alpha" is the name of the parameter.)  Since "elastic net" is more easily recognized than "alpha," I vote for "elasticNetParam."